openedx/core/djangolib/oauth2_retirement_utils.py

```diff
@@ -0,0 +1,41 @@
+from oauth2_provider.models import (
+    AccessToken as DOTAccessToken,
+    Application as DOTApplication,
+    Grant as DOTGrant,
+    RefreshToken as DOTRefreshToken,
+)
+from provider.oauth2.models import (
+    AccessToken as DOPAccessToken,
+    RefreshToken as DOPRefreshToken,
+    Grant as DOPGrant,
+)
+
+def delete_user_from(model, user_id):
```

Comment:
This function is poorly named in my opinion

Reply:
It's intended as a private utility to be used to implement the public-facing methods. It's naming is less consequential. 

Here's what motivated the name:
It's calling convention is: `delete_user_from(model=AccessToken, user=user)`, which I intend to read as "delete user from model AccessToken". Including the parameter label in the function name is a practice I've seen elsewhere.

What do you dislike about it and what would you prefer?